### DISCUSSION TITLE
search auto-complete

### DISCUSSION THREAD

**Discussion by ffbw4hwj** [streamlink/streamlink-twitch-gui#972] [Q&A] [answered]
how do i add or remove items in the auto-complete items of the search field?

**Comment by bastimeyer** [comment, marked as answer]
There is no auto complete function for searching (because there's no API endpoint for it and the main search endpoint is not suitable).

What you're talking about is the search history. It is limited to 50 items and there's also a clear-button above it (trash can icon).

**Comment by ffbw4hwj** [comment]
i meant this one, not the drop down one
https://i.imgur.com/AlIqjWu.png
it shows up when you click the field when it's empty
things in there are what i've searched before but i dont understand what criteria it uses to put things in there, because some of these are from years ago

**Reply by bastimeyer** [reply]
This is not a feature of Streamlink Twitch GUI. Whatever this is, this must come from an external application which hooks into Chromium and the rendered input field.

**Reply by ffbw4hwj** [reply]
ah.. no idea how that happened then
i dont have any mods on streamlink(gui) and i dont have chromium installed elsewhere
